pb/xchain.proto

```diff
@@ -792,3 +793,15 @@ message IdentityAuth {
 message IdentityAuths {
     repeated IdentityAuth auth = 1;
 }
+
+enum ResourceType {
+    CPU = 0;
+    MEMORY = 1;
+    DISK = 2;
+    GAS = 3;
```

Comment:
GAS is too abstract as a ResourceType, cause resource can be converted to gas, the gas is one kind of Resource, looks confusing.  As a symbol of charge paid to the system contract,   equals to `Commission fees`  which we pay for the bank when we create a credit account.  so, I think we can use CFee instead of Gas will be more understandable.

Reply:
XFEE my be a good name